config/index.js

```diff
@@ -44,7 +44,8 @@ const config = {
       apiDate: process.env.API_DATE || 'X-API-Date',
       packageVersion: process.env.PACKAGE_VERSION || 'X-Package-Version',
       nodeVersion: process.env.NODE_VERSION || 'X-Node-Version'
-    }
+    },
+    url_quotes_api: process.env.URL_QUOTES_API
```

Comment:
```suggestion
   quoteApi: {
     url: process.env.URL_QUOTES_API
   }
```

Reply:
It's better if you group variables based on purpose, since in a big project you may have a lot of config variables, so it is better to have them organised. Also, since this properties will be used in code, they should be in `camelCase`